### ISSUE TITLE
Need per-station key-bindings. (Was: Nav control problem: defualt keyboard bindings for SDW keys not manoeuvring ship)

### ISSUE THREAD

**Issue by angrystatic** [smcameron/space-nerds-in-space#149]
For some keyboards the S,D & W keys for the default keyboard controls for the navigation console do not work. In the default keyboard controls A & D yaw the ship left and right. In this bug you could only turn left.

There are other keys within the key bindings that don't seem to function.  No I haven't done a full test. Yes the keyboard works fine outside of the game in the linux desktop (Raspberry PI PIXEL for Mac and PC as well as Linux Mint).

This is likely a simple key bindings problem. But sadly I'm not much of a coder.

**Comment by smcameron** [comment]
Thanks for reporting this.  I have reproduced the problem.  Pretty sure this used to work not long ago, so I must have broken it somewhere along the way.

**Comment by smcameron** [comment]
```
b384b075ab5142d5e6f46fdb2356c2f9882e043c is the first bad commit
commit b384b075ab5142d5e6f46fdb2356c2f9882e043c
Author: Stephen M. Cameron <stephenmcameron@gmail.com>
Date:   Sat Jun 16 08:15:44 2018 -0400

    Add keybindings to switch between various science screen modes
    
    Defaults of d, w, l, s, m, t for Details, Waypoints, Long and Short
    range scan, Mining bot, and Tractor beam, respectively.
    
    Signed-off-by: Stephen M. Cameron <stephenmcameron@gmail.com>

:100644 100644 0511c42f58def00cc6375a6ce318b1e5f0a334ae 62a44670642d5c81216318eb2e93ace50c6f26ba M	snis_client.6
:100644 100644 c4783d0af33f687fbefc93cdb625d774cf6eee00 942136a8763c8235976606c0b62184f1e2a9a9e2 M	snis_client.c
:100644 100644 8daaa8ed6d6b395247fcbbd463228ca6bac629dd bda8c1d5f549b199fbdf60971387db3beb253a37 M	snis_keyboard.c
:100644 100644 1bc55f52a8278d0c63485a0ab7334e305fb6a4f5 78bc4b8e8858856a70ba8c5afc5ddccf1d639ceb M	snis_keyboard.h
```

**Comment by smcameron** [comment]
Temporarily fixed by 42eab179a875b9d9d4c12e7b3b4f28947a6d2336

at the cost of breaking some SCIENCE hotkeys.

The real fix would be to have per-station key bindings.

**Comment by smcameron** [comment]
This should now be fixed by the following commits:

* 76022c313011041b0ed2dfc635b305065fe05166
* e2fae9c65a80efeb7949bb714fc3e469352996e4
